### ISSUE TITLE
[CM11-20140703] [smultron] suspend_backoff wakelock

### ISSUE THREAD

**Issue by Grof87** [LegacyXperia/local_manifests#711]
I'm still having trouble with this wakelock, it disappears if I toggle the cpu governor and reboot, however it appears again after i charge the phone.

https://gist.github.com/Grof87/985c649f59a6fa922ad3

**Comment by zhangziang** [comment]
My device neo always have this issue. it drain my battery.

**Comment by Grof87** [comment]
Whatever was causing suspend_backoff wakelock in my phone is gone. I don't why or how. I tried disabling all the google play services, BT services and even sms push service one by one to see if I could find what was causing the wakelock but no matter what I did it would reappear after a reboot. Tried messing with performance settings, it didn't help also. And now, for the last 24 hours, that wakelock is gone. I tried charging, rebooting, enabling/disabling all the services again just to see if it would appear again but it hasn't. I've even set my governor to ondemand and ticked apply at boot but the wakelock is not appearing. I'll report back if anything changes.
